assets/scss/_home.scss

```diff
@@ -0,0 +1,3 @@
+#omicsarea-tree tr{
+  background: white;
```

Comment:
Why is this necessary actually?

Reply:
The tree creation function creates an HTML table. We use foundation for the style and it automatically applies its style in all tables and therefore the tree. Without this CSS, we'd have gray bars in the background.